packages-web/maps/src/components/GoogleMap.ts

```diff
@@ -172,7 +178,12 @@ export class GoogleMap extends Component<GoogleMapsProps, GoogleMapState> {
                 if (currentLocation.locationAttr && currentLocation.locationAttr.onClickEvent !== "doNothing") {
                     marker.addListener("click", (event: google.maps.MouseEvent) => {
                         if (this.props.onClickMarker && currentLocation.locationAttr) {
-                            this.props.onClickMarker(event, currentLocation.locationAttr);
+                            const options = {
+                                options: {
+                                    GUID: currentLocation.mxObject ? currentLocation.mxObject.getGuid() : undefined
```

Comment:
Can there be markers without a Guid?

Reply:
Static markers are not possible to click